e3sm_diags/driver/zonal_mean_2d_driver.py

```diff
@@ -161,7 +161,7 @@ def _convert_g_kg_to_ppm_units(
             parameter.current_set == "zonal_mean_2d_stratosphere"
             and parameter.var_id == "Q"
         ):
-            ds_new[var_key] = ds_new[var_key] * 1000.0
+            ds_new[var_key] = ds_new[var_key] * 28.97 / 18.0 * 1000.0
             ds_new[var_key].attrs["units"] = "ppm"
```

Comment:
The new units should be "ppmv" with "v" indicating ppm by volume.

Reply:
Thanks for the catch, just updated.